.github/workflows/run-pytest.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: [3.6, 3.9]
+        python-version: [3.7, 3.11]
```

Comment:
🎉

Reply:
We should transition this to 3.8 right?

Reply:
It's fine to leave 3.7
I have deprecated 3.6, because in 3.6 I can't import typing in usual way. Some of the programs are using ubiquerg with older version, so I would say let's keep 3.7